massa-pos-exports/src/controller_traits.rs

```diff
@@ -62,7 +60,6 @@ pub trait SelectorController: Send + Sync {
     /// Get every [Selection]
     ///
     /// Only used in tests for post-bootstrap selection matching.
-    #[cfg(feature = "testing")]
```

Comment:
Why did you removed the constraint ?

Reply:
Because of https://github.com/massalabs/massa/issues/3016(to be fixed in a follow-up)

Reply:
Did you tried with the comment I made in the issue ? I think it solves the problem.

Reply:
I haven't tried, and I think it's best done in a follow-up, so that this PR is not blocked by something unrelated.